renovate.json

```diff
@@ -649,6 +649,23 @@
         "Team:Monitoring"
       ],
       "enabled": true
+    },
+    {
+      "groupName": "AlertingEmails",
+      "matchPackageNames": [
+        "nodemailer"
+      ],
+      "reviewers": [
+        "team:response-ops"
+      ],
+      "matchBaseBranches": [
+        "main"
+      ],
+      "labels": [
+        "release_note:skip",
+        "backport:prev-minor"
+      ],
+      "enabled": true
```

Comment:
Hi @elastic/response-ops! Since nodemailer is a top level dependency used heavily within plugins owned by your team, how do you feel about this entry to renovate bot to be tagged in future changes?

Reply:
Ping @elastic/response-ops

Reply:
@kobelb @pmuellr Heya! Are you able to give us some guidance here?

Reply:
I assume this means for any renovate/etc PRs generated for this package, we'll get tagged to review?  

Makes sense.

I'll try this locally, but could you add the tag to have it create a cloud (stateful, not serverless) deployment we can test with?